config/airseeker.example.json

```diff
@@ -25,5 +25,7 @@
   "deviationThresholdCoefficient": 1,
   "signedDataFetchInterval": 10,
   "signedApiUrls": [],
-  "walletDerivationScheme": { "type": "managed" }
+  "walletDerivationScheme": { "type": "managed" },
+  "stage": "dev",
+  "version": "2.2.1"
```

Comment:
Can we add `version` and `stage` fields to heartbeat payload? If you don't mind, it would be better to have `deploymentTimestamp` either 😄 . Not urgent.

Reply:
Sure, I was thinking that they are a bit unnecessary because there is the config hash. But yeah, you won't know what's the stage and version in Grafana logs without logging them in raw form. I'll change this.